flamedisx/utils.py

```diff
@@ -279,3 +280,17 @@ def load_config(config_files=None):
             if not k.startswith('_')})
 
     return config
+
+
+# Taken from straxen to filter arguments for interpolators
+@export
+def filter_kwargs(func, kwargs):
```

Comment:
There is a similar function for the sources `_filter_source_kwargs`, but it is good to have a general function for that.